docs/topics/publishing.md

```diff
@@ -83,8 +83,8 @@ Remove locally created artifacts and use the downloaded ones
 
 ```
 $ make clean
-$ cp artifacts/WORKFLOW_RUN_NUMBER/DockerImages/* artifacts/
-$ make test SUITE=<suite-name>
+$ cp artifacts/WORKFLOW_RUN_NUMBER/BuildArtifacts/* artifacts/
+$ ./test.sh <suite-name>
```

Comment:
This is the only usage of `BuildArtifacts` I can find, but `DockerImages` is still used in quite a few places - is the splitting deliberate, or should there be more or fewer changes?

Reply:
Good catch, and interesting. That should definitely be `DockeImages`, as it is on `main` in this doc. Will update and look around as to why git is preferring this to what is on `main``, I didn't change the file directly but Robert and I were working off the same base branch of his "repro builds" before it was merged down so I'm thinking this got missed in a merge resolution.

Reply:
Ok, I diff'd with main and this is the only thing different in that doc.